src/components/NcAppContentDetails/NcAppContentDetails.vue

```diff
@@ -5,13 +5,18 @@
 
 <template>
 	<div class="app-content-details">
-		<!-- @slot Provide content to the details view -->
 		<slot />
 	</div>
 </template>
 
-<script>
-export default {
-	name: 'NcAppContentDetails',
-}
+<script setup lang="ts">
+import type { Slot } from 'vue'
+
+defineSlots<{
+	/**
+	 * Provide content to the details view.
+	 * Passing content to this slot is required!
+	 */
+	default: Slot
```

Comment:
```suggestion
	default?: Slot
```

Reply:
But this time its required

Reply:
Vue does not support required slots, and this type doesn't define it. It is a type representing `context.slots` object.

Reply:
> Vue does not support required slots, and this type doesn't define it. It is a type representing context.slots object.

But mostly this is describing it (as the object then needs to be available